index.html

```diff
@@ -60,6 +58,87 @@ <h2 class="title text-primary">I’m a software developer</h2>
                 <img src="assets/images/Disabled.svg" alt="">
             </div>
         </section>
+        <section id="my-recent-work">
+            <h2 class="title 2">My Recent<br>Work<br></h2>
+                <div class="indicator"></div>
+                <div class="img-patholder">
+                    <div class="white-box">
+                        <div class="title 2">Multi-Post Stories <br>Gain+Glory</div>
```

Comment:
- [ ] Kindly notice that there is a slight discrepancy in the `color` of the font between the Figma reference and your design. please find the correct font `color` for this text. 

|Figma Reference|Your Design|
-|-
|![image](https://user-images.githubusercontent.com/18730106/153637141-9be5ec64-ca28-4f03-b30b-d9e07c91d7e6.png)|![image](https://user-images.githubusercontent.com/18730106/153637160-c073feb5-677d-4432-a957-c4c37d28fe3a.png)|